Adyen/UI/Form/Items/Value Pickers/Issuer List Picker/FormIssuerPickerItem.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+@_spi(AdyenInternal)
+public typealias IssuerPickerItem = BasePickerElement<Issuer>
+
+@_spi(AdyenInternal)
+public final class FormIssuersPickerItem: BaseFormPickerItem<Issuer> {
```

Comment:
I think this class is redundant, can we just use `BaseFormPickerItem<Issuer> ` directly and make a typealias from `BaseFormPickerItem<Issuer>` to  `FormIssuersPickerItem `

Reply:
BaseFormPickerItem is an Abstract class that needs to be subclassed.

Reply:
is it abstract? I don't think its abstract

Reply:
It is a open class that needs to be subclassed. We are subclassing it for **FormPhoneExtensionPickerItem**  and **FormRegionPickerItem**